identifiers/country-in/special.csv

```diff
@@ -0,0 +1,19 @@
+id,name

```

Comment:
I think naming this file `vidhan_parishad_special.csv` or `vidhan_parishad_special_constituencies.csv` would make it clearer this is specifically for Vidhan Parishad

Reply:
Done.